Dockerfile

```diff
@@ -86,6 +86,18 @@ RUN --mount=target=. \
     GIT_TAG=${GIT_TAG} \
     make BINARY=/out/docker  -f builder.Makefile cross
 
+FROM base AS make-cross-single
+ARG BUILD_TAGS
+ARG GIT_TAG
+ARG TARGETOS
+ARG TARGETARCH
+RUN --mount=target=. \
+    --mount=type=cache,target=/go/pkg/mod \
+    --mount=type=cache,target=/root/.cache/go-build \
+    BUILD_TAGS=${BUILD_TAGS} \
+    GIT_TAG=${GIT_TAG} \
+    make BINARY=/out/docker -f builder.Makefile cross-${TARGETOS}-${TARGETARCH}
```

Comment:
could we avoid duplicating all this, and reuse the make-cross ? (adding optional ARGS for TARGETOS and TARGETARCH maybe ? )

Reply:
`make cross` currently builds all the cross platforms so I'm not sure this'd be easy. By default, TARGETOS and TARGETARCH are set to the builder's value.